modules/cnf-configuring-cvl-nic-as-oc.adoc

```diff
@@ -37,9 +37,9 @@ spec:
   - name: "slave"
     interface: <interface_name> <1>
     ptp4lOpts: "-2 --summary_interval -4" <2>
-    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"
-    ptpSchedulingPolicy: "SCHED_FIFO" <3>
-    ptpSchedulingPriority: 65 <4>
+    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16" <3>
```

Comment:
The changes in this module were not showing up for me in the preview. Then I noticed that this module is not in the assembly that is listed in line 3
https://github.com/openshift/openshift-docs/blob/main/networking/using-ptp.adoc
This shows why the preview isn't rendering.

Is that right @aireilly or is there something I'm missing here? Or maybe the module accidentally get removed at some point